code/modules/admin/verbs/adminevents.dm

```diff
@@ -386,3 +386,30 @@
 	log_admin("[key_name(usr)] removed mob ability [ability_name] from mob [marked_mob].")
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Remove Mob Ability") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/command_report_footnote()
+	set category = "Admin.Events"
+	set name = "Command Report Footnote"
+	set desc = "Adds a footnote to the roundstart command report."
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	var/datum/command_footnote/command_report_footnote = new /datum/command_footnote()
+
+	command_report_footnote.message = tgui_input_text(usr, "This message will be attached to the bottom of the roundstart threat report. It will likely not be seen if the report has already been sent.", "P.S.")
+
+	if(!command_report_footnote.message)
+		return
+
+	command_report_footnote.signature = tgui_input_text(usr, "Whose signature will appear on this footnote?", "Also sign here, here, aaand here.")
+
+	if(!command_report_footnote.signature)
+		command_report_footnote.signature = "Anonymous"
```

Comment:
what if this said classified instead of anonymous, that would be pretty funny i think